packages/graphql/src/classes/Node.ts

```diff
@@ -202,6 +207,13 @@ class Node extends GraphElement {
         };
     }
 
+    public get aggregateTypeNames(): AggregateTypeNames {
+        return {
+            selection: `${this.name}AggregateSelection`,
+            input: `${this.name}AggregateSelectionInput`,
```

Comment:
This should probably use the pascal case singular for consistency - for instance we would want `AWSInstance` to have `AwsInstanceAggregateSelection` like we do elsewhere... I think. 😬 

```suggestion
            selection: `${this.pascalCaseSingular}AggregateSelection`,
            input: `${this.pascalCaseSingular}AggregateSelectionInput`,
```

Reply:
We should however I think doing so could cause unforeseen breaking changes to users with unusual type names such as the one demonstrated in this issue `SICIndustry`.

Reply:
Shall we just do it? Thoughts @angrykoala and @tbwiss?

Reply:
I vote for consistency in this case. so pro pascalCaseSingular. but yes we might get some bug reports due to this breaking change.